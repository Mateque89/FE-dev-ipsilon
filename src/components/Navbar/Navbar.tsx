import React from 'react'
import AppBar from '@mui/material/AppBar';
import Box from '@mui/material/Box';
import Toolbar from '@mui/material/Toolbar';
import Typography from '@mui/material/Typography';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faVirus } from '@fortawesome/free-solid-svg-icons'


function Navbar() {
    return (
        <Box sx={{ flexGrow: 1 }}>
            <AppBar position="static">
                <Toolbar>
                    <Box sx={{ mr: 2 }} >
                        <FontAwesomeIcon icon={faVirus} size="2x" />
                    </Box>
                    <Typography variant="h6" sx={{ flexGrow: 1, fontWeight: 'bold' }}>
                        COVID19-stats
                    </Typography>
                </Toolbar>
            </AppBar>
        </Box>
    )
}
export default Navbar;