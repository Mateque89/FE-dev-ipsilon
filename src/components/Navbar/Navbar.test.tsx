import React from 'react';
import { create } from 'react-test-renderer'
import Navbar from './Navbar';

it('Navbar renders correctly', () => {
    const tree = create(<Navbar />).toJSON();
    expect(tree).toMatchSnapshot();
});
