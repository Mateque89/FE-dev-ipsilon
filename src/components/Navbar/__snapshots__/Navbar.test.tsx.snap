// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<div
  className="MuiBox-root css-i9gxme"
>
  <header
    className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-hip9hq-MuiPaper-root-MuiAppBar-root"
  >
    <div
      className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-1t29gy6-MuiToolbar-root"
    >
      <div
        className="MuiBox-root css-1ay9vb9"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-virus fa-w-16 fa-2x "
          data-icon="virus"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M483.55,227.55H462c-50.68,0-76.07-61.27-40.23-97.11L437,115.19A28.44,28.44,0,0,0,396.8,75L381.56,90.22c-35.84,35.83-97.11,10.45-97.11-40.23V28.44a28.45,28.45,0,0,0-56.9,0V50c0,50.68-61.27,76.06-97.11,40.23L115.2,75A28.44,28.44,0,0,0,75,115.19l15.25,15.25c35.84,35.84,10.45,97.11-40.23,97.11H28.45a28.45,28.45,0,1,0,0,56.89H50c50.68,0,76.07,61.28,40.23,97.12L75,396.8A28.45,28.45,0,0,0,115.2,437l15.24-15.25c35.84-35.84,97.11-10.45,97.11,40.23v21.54a28.45,28.45,0,0,0,56.9,0V462c0-50.68,61.27-76.07,97.11-40.23L396.8,437A28.45,28.45,0,0,0,437,396.8l-15.25-15.24c-35.84-35.84-10.45-97.12,40.23-97.12h21.54a28.45,28.45,0,1,0,0-56.89ZM224,272a48,48,0,1,1,48-48A48,48,0,0,1,224,272Zm80,56a24,24,0,1,1,24-24A24,24,0,0,1,304,328Z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
      </div>
      <h6
        className="MuiTypography-root MuiTypography-h6 css-19xrsc3-MuiTypography-root"
      >
        COVID19-stats
      </h6>
    </div>
  </header>
</div>
`;
